app/lib/filters/show_all_applications.rb

```diff
@@ -0,0 +1,23 @@
+module Filters
+  class ShowAllApplications < Base
+    def apply
+      if show_all_applications.empty?
```

Comment:
```suggestion
      unless show_all_applications.include?("show_all")
```

This value `show_all` comes from the value of the checkbox: https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/2078/files#diff-14c048dcf48c873d6deb7ce9a22b554330cfadcac4a8e1c805b480f593b15983R69

I think your solution works too but I prefer this as it's more explicit and allows us to add more checkboxes in the future.